messages/observations/ground_velocity.json

```diff
@@ -24,6 +23,16 @@
             "title": "Speed over ground [knots]",
             "description": "Reported speed over ground."
         },
+        "sog_transverse_bow": {
```

Comment:
If you had used the location object, there could have been two ground velocity observations, one for the aft and one for the bow. Then this fields (`sog_transverse_bow` and `sog_transverse_stern`) wouldn't have be necessary. I know we talked about it, but I don't remember what we concluded. Wouldn't it be better to calculate these transverse speeds in the frontend?

Reply:
Not sure we concluded anything. But my general standpoint is to not send redundant/derived data, i.e. any derived speeds/velocities should be calculated in the frontend.

Reply:
Also, this message in combination with https://github.com/MO-RISE/brefv/blob/main/messages/observations/rate_of_turn.json should make dual ground velocity observations redundant as well?